src/pages/HomePage/useBlockListData.ts

```diff
@@ -35,11 +36,16 @@ const useBlockListData = (currentPageNumber: number) => {
 
       manualFetch && setManualLoading(true)
 
-      const { data } = await client.blocks.getBlocks({ page: pageNumber, limit: 8 })
+      try {
+        const { data } = await client.blocks.getBlocks({ page: pageNumber, limit: 8 })
 
-      if (data) {
-        console.log('Number of block fetched: ' + data.blocks?.length)
-        setBlockList(data)
+        if (data) {
+          console.log('Number of block fetched: ' + data.blocks?.length)
+          setBlockList(data)
+          setPage(pageNumber)
+        }
+      } catch (e) {
+        setPage(1)
```

Comment:
This was the only API query that was not wrapped in a `try/catch`, the rest of the pages work fine